programs/uxd/Cargo.toml

```diff
@@ -21,6 +21,7 @@ no-log-ix-name = []
 cpi = ["no-entrypoint"]
 test-bpf = []
 default = [
+  "cpi",
```

Comment:
Might add as comment "for recursive program calls" till we forget x)

Reply:
done!